cfg/config.go

```diff
@@ -67,17 +67,18 @@ type SnapshotConfiguration struct {
 }
 
 type NATSConfiguration struct {
-	URLs             []string `toml:"urls"`
-	SubjectPrefix    string   `toml:"subject_prefix"`
-	StreamPrefix     string   `toml:"stream_prefix"`
-	ServerConfigFile string   `toml:"server_config"`
-	SeedFile         string   `toml:"seed_file"`
-	CredsUser        string   `toml:"user_name"`
-	CredsPassword    string   `toml:"user_password"`
-	CAFile           string   `toml:"ca_file"`
-	CertFile         string   `toml:"cert_file"`
-	KeyFile          string   `toml:"key_file"`
-	BindAddress      string   `toml:"bind_address"`
+	URLs                    []string `toml:"urls"`
+	SubjectPrefix           string   `toml:"subject_prefix"`
+	StreamPrefix            string   `toml:"stream_prefix"`
+	ServerConfigFile        string   `toml:"server_config"`
+	SeedFile                string   `toml:"seed_file"`
+	CredsUser               string   `toml:"user_name"`
+	CredsPassword           string   `toml:"user_password"`
+	CAFile                  string   `toml:"ca_file"`
+	CertFile                string   `toml:"cert_file"`
+	KeyFile                 string   `toml:"key_file"`
+	BindAddress             string   `toml:"bind_address"`
+	ConnectTotalWaitMinutes int      `toml:"connect_total_wait_minutes"`
```

Comment:
I will have it consistent with other time units I believe everywhere it's seconds so this should be seconds too. We should rename it to something like ConnectionTimeout I believe

Reply:
I've updated to `ConnectRetries` as I thought that made more sense. Happy to rename it again as you see fit.